### PR TITLE
CBA_settings frequencies optimisation

### PR DESCRIPTION
**Pull request by Dorbedo**
**When merged this pull request will:**
- change the channelamount from a fixed amount into a min amount
- add cba frequencies settings input restrictions

### CODE REVIEW

File: addons/core/stringtable.xml [hunk 1]
```diff
@@ -1556,10 +1556,10 @@
                 <Japanese>長距離無線機の標準設定 (NATO):</Japanese>
             </Key>
             <Key ID="STR_TFAR_SETTINGS_DefaultRadioFrequencies_LR_west_desc">
-                <Original>Define the default LR-settings for the NATO side as a list. e.g.: 99,101,250</Original>
-                <English>Define the default LR-settings for the NATO side as a list. e.g.: 99,101,250</English>
-                <German>Das Festlegen der Standard Langwellenfrequenzen für die NATO-Streitkräfte erfolgt als Liste. z.B.: 99,101,250</German>
-                <Japanese>NATO 側で使われる長距離無線機の標準設定をリストで定義します。(例: 99, 101, 250)</Japanese>
+                <Original>Define the default LR-settings for the NATO side as a list. e.g.: 45, 47, 70</Original>
```
Comment (dedmen) [inline addons/core/stringtable.xml]: why that?

Comment (Dorbedo) [reply addons/core/stringtable.xml]: I've just lowered the examples for the LR-frequencies.
Everything higher than 87 is not a valid LR frequency.

File: addons/core/functions/fnc_settingForceArray.sqf [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    Name: TFAR_fnc_settingForceArray
+
+    Author(s):
+        Dorbedo
+
+    Description:
+        Forces the input into an array format and removes invalid input
+
+    Parameters:
+    0: STRING - frequencies input
```
Comment (dedmen) [inline addons/core/functions/fnc_settingForceArray.sqf]: Indentation wrong. If you don't care I'll push anyway. Just trying to trigger your OCD.

Comment (Dorbedo) [reply addons/core/functions/fnc_settingForceArray.sqf]: well done.... next time trigger your own

**Pull request merged**